guides/v2.1/security/google-recaptcha.md

```diff
@@ -0,0 +1,47 @@
+---
+group: security
+title: Google reCAPTCHA
+functional_areas:
+  - Configuration
+---
+
+This extension adds the Google reCAPTCHA module to your Magento installation. Google reCAPTCHA provides a greater level of security for both the storefront and Admin than what is available with the standard CAPTCHA, and gives you the ability to:
+
+- Verify when customers create accounts, retrieve passwords, log in to their accounts, or contact your company.
+- Enhance security when Admin users log in.
+
+Google reCAPTCHA reduces potential user error when entering a series of letters and numbers, and encourages cart conversion without adding hurdles during checkout.
+
```

Comment:
I'm not sure what this means:  "user error when entering a series of letters and numbers"

Reply:
Basically a captcha code, edited for clarity